sdk/communication/azure-communication-identity/tests.yml

```diff
@@ -1,7 +1,42 @@
 trigger: none
 
 stages:
-  - template: /sdk/communication/communication-tests-template.yml
-    parameters:
-      PackageName: azure-communication-identity
-      SafeName: azurecommunicationidentity
+    - template: /eng/pipelines/templates/stages/archetype-sdk-tests.yml
+      parameters:
+          Clouds: 'Public,Int'
+          TestMode: 'LIVE'
+          ServiceDirectory: communication
+          EnvVars:
+              SKIP_LIVE_TEST: TRUE
+          MatrixReplace:
+              - TestSamples=.*/true
+          CloudConfig:
+              Public:
+                  SubscriptionConfigurations:
+                      - $(sub-config-azure-cloud-test-resources)
+                      - $(sub-config-communication-services-cloud-test-resources-common)
+                      - $(sub-config-communication-services-cloud-test-resources-java)
+              Int:
```

Comment:
```suggestion
              PPE:
```

Reply:
I think it might be better to support some sort of `CloudConfig` parameter in the `communication-tests-template.yml` that defaults to the public+int values, instead of duplicating everything here? I can help more with some examples/suggestions next week.

Reply:
@benbp replacing Int with PPE causes tests to be skipped.

I totally agree that we shouldn't repeat code. Could you please help with correct syntax for default variables?